_pages/algorithmia-enterprise/insights.md

```diff
@@ -0,0 +1,65 @@
+
+## One-time administrator setup
+
+The first step in allowing Insights to be collected and reported for algorithms is to have the Algorithmia Platform Administrator connect your cluster to a Kafka broker that your organization operates to receive metrics created by the Algorithmia platform.
+
+Navigate to the _Insights_ administration page, which can be found under the "System Actions" section.
+
+**Screenshot and list of credentials needed TBD pending UX completion - https://algorithmia.atlassian.net/browse/INSIGHTS-31**
+
+From here, enter the information about your Kafka broker. This includes a URL and credentials necessary to connect. The Algorithmia platform supports [SCRAM](https://en.wikipedia.org/wiki/Salted_Challenge_Response_Authentication_Mechanism) for authentication; please see the [Kafka SCRAM setup guide](https://kafka.apache.org/documentation/#security_sasl_scram) for creating credentials that the Algorithmia platform will use to connect.
+
+## Using Insights in an algorithm
+
+Each algorithm in your Algorithmia cluster has independent settings for whether to collect and report Insights. Additionally, each published version of your algorithm can have Insights turned on or off.
+
+### Creating inference-related metrics for your algorithm
+
+Algorithmia [client libraries](../clients) are used to report inference-related metrics that you create inside of your algorithm code. Please see the "Publishing Insights" section of the appropriate [client library guide](../clients) for the language that you're using.
+
+### Enabling Insights for an algorithm version
+
+Enabling Insights for an algorithm version happens during [algorithm publishing](../../algorithm-development/your-first-algo/#publish-your-algorithm).
+
+When using the web UI publishing workflow, a toggle will appear to enable Insights for the specific algorithm version that you are publishing.
+
+**Screenshot pending UX completion - https://algorithmia.atlassian.net/browse/INSIGHTS-31**
+
+When using the HTTP API or client libraries to publish your algorithm, passing the key value pair `"insights_enabled":true` in the `settings` hash will enable Insights for your algorithm. For example, when using [cURL](../../clients/curl):
+
+{% highlight bash %}
+curl -H 'Authorization: Simple YOUR_API_KEY' https://<algorithmia-cluster-host>/v1/algorithms/<algorithm-owner>/<algorithm-name>/version --data-binary '{"settings":{"insights_enabled":true}}'
+{% endhighlight %}
+
+One you've published your algorithm with Insights enabled, all executions of that specific version of that algorithm will trigger your Insights to be collected and reported to the Kafka broker that your Algorithmia Platform Administrator has set up.
```

Comment:
Replace "One" with "Once".
Double "that" usage with "that specific version of that algorithm". Could we change to "that specific version of the algorithm"?